pkg/controller/activegate/activegate_create.go

```diff
@@ -11,18 +12,56 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/dtclient"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+func (r *ReconcileActiveGate) createCustomPropertiesConfigMap(kubernetesMonitoringSpec *dynatracev1alpha1.KubernetesMonitoringSpec) error {
+	if kubernetesMonitoringSpec.Enabled &&
+		kubernetesMonitoringSpec.CustomProperties != nil &&
+		kubernetesMonitoringSpec.CustomProperties.Value != "" &&
+		kubernetesMonitoringSpec.CustomProperties.ValueFrom == "" {
+
+		configMap := &corev1.ConfigMap{}
+		err := r.client.Get(context.TODO(), client.ObjectKey{
+			Namespace: _const.DynatraceNamespace,
+			Name:      _const.CustomPropertiesConfigMapName,
+		}, configMap)
+		if err != nil {
+			if k8serrors.IsNotFound(err) {
+				configMap = &corev1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      _const.CustomPropertiesConfigMapName,
```

Comment:
You can have more than one DynaKube object, requiring more than one config map. You'd also need to detect changes if the field on DynaKube changes.